app/src/main/java/com/codepath/apps/critterfinder/activities/PetBrowserActivity.java

```diff
@@ -104,7 +104,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (requestCode == SEARCH_FILTER_REQUEST_CODE) {
             mSearchFilter = Parcels.unwrap(data.getParcelableExtra(PetSearchFilterActivity.EXTRA_SEARCH_FILTER));
             Snackbar.make(findViewById(android.R.id.content), "Search filters have been updated. Gender: " + mSearchFilter.getGender().toString(), Snackbar.LENGTH_LONG).show();
-            // TODO - refresh our search?
+            mSwipeablePetsFragment.doPetSearch(mSearchFilter);
```

Comment:
sweet. We can probably kill the debug toast I had hear as well. Since we're performing a search now.